### PR TITLE
inclusive naming for kubernetes-master

### PR DESCRIPTION
**Pull request by addyess**
the charm kubernetes-master is being renamed to kubernetes-control-plane and is available on [charmhub.io](https://charmhub.io/kubernetes-control-plane)

The following PR updates the charm name as well as uses associated with the integration testing.  It doesn't address all inclusive naming issues within the charm itself -- but just attempts to alter the name of the charm for maximizing backwards compatability

### CODE REVIEW

File: tests/integration/conftest.py [hunk 1]
```diff
@@ -35,22 +20,24 @@ async def kubernetes(ops_test):
...
     )
     if retcode != 0:
         log.error(f"retcode: {retcode}")
         log.error(f"stdout:\n{stdout.strip()}")
         log.error(f"stderr:\n{stderr.strip()}")
-        pytest.fail("Failed to copy kubeconfig from kubernetes-master")
+        pytest.fail("Failed to copy kubeconfig from kubernetes-control-plane")
 
     namespace = (
         "test-kubernetes-master-integration-"
```
Comment (stonepreston) [inline tests/integration/conftest.py:33]:
```suggestion
        "test-kubernetes-control-plane-integration-"
```

File: README.md [hunk 2]
```diff
@@ -32,7 +33,7 @@ examples of complete models of Kubernetes clusters.
 
 For full install instructions, please see the [Charmed Kubernetes documentation](https://ubuntu.com/kubernetes/docs/quickstart).
 
-For details on configuring and operating this charm, see the [kubernetes-master documentation](https://ubuntu.com/kubernetes/docs/charm-kubernetes-master) on the same site.
+For details on configuring and operating this charm, see the [kubernetes-contro-plane documentation](https://ubuntu.com/kubernetes/docs/charm-kubernetes-master) on the same site.
```
Comment (stonepreston) [inline README.md:36]:
```suggestion
For details on configuring and operating this charm, see the [kubernetes-control-plane documentation](https://ubuntu.com/kubernetes/docs/charm-kubernetes-master) on the same site.
```

**Review by stonepreston — COMMENTED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -32,7 +33,7 @@ examples of complete models of Kubernetes clusters.
 
 For full install instructions, please see the [Charmed Kubernetes documentation](https://ubuntu.com/kubernetes/docs/quickstart).
 
-For details on configuring and operating this charm, see the [kubernetes-master documentation](https://ubuntu.com/kubernetes/docs/charm-kubernetes-master) on the same site.
+For details on configuring and operating this charm, see the [kubernetes-control-plane documentation](https://ubuntu.com/kubernetes/docs/charm-kubernetes-master) on the same site.
```
Comment (Cynerva) [inline README.md:36]: @evilnick Just bringing this to your attention as a URL that may need to be updated later.

**Review by Cynerva — APPROVED** [review]

**Pull request merged**